src/components/forms/dialog-forms/add-new-card-dialog-form/add-new-card-dialog-form.stories.tsx

```diff
@@ -16,6 +16,8 @@ type Story = StoryObj<typeof meta>
 
 export const AddNewCardFormExample: Story = {
   args: {
+    action: 'CREATE',
+    deckId: 'cly7c2vqa0drxpb015rp9sbi7',
     onOpenChange: action('onOpenChange action invoked!'),
     onSubmit: action('onSubmit action invoked!'),
     open: true,
```

Comment:
История не работает:
![image](https://github.com/atpradical/flash-cards-team/assets/141486338/5d8942bc-589a-4783-87c1-1b0db5063e94)

Предлагаю добавить после args:
```suggestion
render: () => (
    <Provider store={store}>
      <AddNewCardDialogForm {...AddNewCardFormExample.args} />
    </Provider>
  ),
```

Reply:
fixed